rbac/server/api/proposals.py

```diff
@@ -192,5 +191,13 @@
     else:
         user_resource = await fetch_user_resource(conn, proposal_resource.get("object"))
         proposal_resource["approvers"] = [user_resource.get("manager")]
+    i = 0
+    approvers_count = len(proposal_resource["approvers"])
+    while i < approvers_count:
+        user_resource = await users_query.fetch_manager_chain(
```

Comment:
Calling the returned manager chain "user_resource" here, isn't too readable, since it's actually a list of IDs. (vs a few lines above, where user_resource is really a user_resource, with all their associated info.)